Makefile

```diff
@@ -135,9 +139,17 @@ generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs Go related generate tar
 		--input-dirs=./api/v1alpha2 \
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
+	$(CONTROLLER_GEN) \
+			object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+			paths=./cmd/clusterctl/api/...
+	## generate code for embedded clusterctl api manifest
+	mkdir -p ./cmd/clusterctl/config/manifest/
+	kustomize build ./cmd/clusterctl/config/crd > ./cmd/clusterctl/config/manifest/clusterctl-api.yaml
+	go generate ./cmd/clusterctl/config/generate.go
+	rm -r ./cmd/clusterctl/config/manifest/
 
 .PHONY: generate-manifests
-generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
+generate-manifests: $(CONTROLLER_GEN) $(GOBINDATA_GEN) ## Generate manifests e.g. CRD, RBAC etc.
```

Comment:
Should the `$(GOBINDATA_GEN) dependency be added to the `generate-go` target instead? This target doesn't appear to be using go-bindata

Reply:
yes!